collector/consumer/processor/kindlingformatprocessor/protocol.go

```diff
@@ -74,6 +80,17 @@ func fillTopologyHttpProtocolLabel(g *gauges) {
 	g.targetLabels.AddStringValue(constlabels.StatusCode, strconv.FormatInt(g.Labels.GetIntValue(constlabels.HttpStatusCode), 10))
 }
 
+func fillSpanHttpProtocolLabel(g *gauges) {
+	g.targetLabels.AddStringValue("http.method", g.Labels.GetStringValue(constlabels.HttpMethod))
+	g.targetLabels.AddStringValue("http.endpoint", g.Labels.GetStringValue(constlabels.HttpUrl))
+	g.targetLabels.AddIntValue("http.status_code", g.Labels.GetIntValue(constlabels.HttpStatusCode))
+	// TODO trace_id and trace_type
+	//kv = append(kv, attribute.String("http.trace_id", s.Protocol.HTTP.TraceID))
+	//kv = append(kv, attribute.String("http.trace_type", s.Protocol.HTTP.TraceType))
```

Comment:
We have had these two fields after the `NetworkAnalyzer` parses them. So just add them.
<img width="422" alt="image" src="https://user-images.githubusercontent.com/46807570/156491499-f445fbe7-4c2c-43f6-bcac-897a5cd29828.png">
<img width="697" alt="image" src="https://user-images.githubusercontent.com/46807570/156491601-0d1aa64c-de7e-4804-ba78-f0954bb2c528.png">

Reply:
Changed already, take a look

Reply:
Changed already, take a look